helm/cluster-azure/templates/list.yaml

```diff
@@ -3,6 +3,20 @@
 {{- include "psps" . }}
 {{- end }}
 ---
+{{- include "helmrepository-default" . }}
+---
+{{- include "cni-driver-helm-release" . }}
+---
+{{- include "cloud-controller-manager-helm-release" . }}
+---
+{{- include "cloud-node-manager-helm-release" . }}
+---
+{{- include "csi-driver-helm-release" . }}
+---
+{{- include "coredns-helm-release" . }}
+---
+{{- include "helmreleases-cleanup-hook" . }}
+---
```

Comment:
is there any advantage defining the templates and including them here specifically rather than just creating normal helm templates files?

Reply:
not really ( at least that i can think about ) , but since all the other resources for cluster-azure were already handled this way i think it is better to keep the same format 

this was done at the start of the chart https://github.com/giantswarm/cluster-azure/pull/3 :shrug:

Reply:
@bavarianbidi any reason why we used the `list.yaml` ?

Reply:
I think we used the `list.yaml` at first because we needed some of the resources to be applied *in order*, or it wouldn't work. And the only way to achieve ordering was by explicitly including the templates.

Reply:
I see, make sense.

Well given that soon we are handing over this to the `cloud integration team` i will leave it as it is for now and the team can decide what to do with it :+1: 

sounds good ?

Reply:
works for me :+1:  although I think having cluster apps as similar as possible is always better

Reply:
totally agree on that :+1:  

I am just not sure if right now is the time for me to change that , but depending on time before the handover to cloud integration team happens i might be able to do that

Reply:
oh i see, `cluster-aws` has __the list__ too but you added the templates individually. 

Sure can do :+1: 

will do it before i merge this

Reply:
this is done